pkg-tests/.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "rules": {
+    "flowtype/require-valid-file-annotation": "off",
+    "flowtype/require-return-type": "off"
+  }
+}
```

Comment:
Disabled flow (in this directory only) for the time being, the utils don't really need typing right now (especially since they're by definition covered by the tests :).

Reply:
You are probably right but I think we should generally try to use flow types across the board. It seems that you could add flow types to everything rather quickly – do you mind spending some time on this? It will make it significantly easier for other people to work on this code, for example in Nuclide, where we have type definitions available and auto-completion.

Reply:
Also helps me review the code ;)